### DISCUSSION TITLE
Add option to add Action button that redirects you back to index after form submit

### DISCUSSION THREAD

**Discussion by martinsjek** [filamentphp/filament#7862] [Ideas]
I think this would be cool if we could "Submit and close" form after we are done editing it.
We already have "Create & Create another", it would be great to have "Create & close" option as well.

**Comment by danharrin** [comment]
You can add this to `getFormActions()` on the page yourself if you want, it's unlikely it will be a core feature any time soon as it is easy to accomplish in userland

**Reply by martinsjek** [reply]
Well in that case I would need to override CreateRecord class in case if I want to use it in multiple places.
It would be great if the core feature would be like $canCreateAnother.
